src/components/core/DpBulkEditHeader.vue

```diff
@@ -38,6 +40,10 @@ export default {
   computed: {
     selectedItemsText () {
       return this.selectedItemsCount + ' ' + this.selectionText
+    },
+
+    unselect () {
+      return de.operations.deselect.element
```

Comment:
I'd call the computed `deselect` as well.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/9a6ce2821e733a9d19fa5c3bf4e5b0e8eca82c70